anta/input_models/routing/bgp.py

```diff
@@ -68,8 +72,11 @@ class BgpAddressFamily(BaseModel):
     check_peer_state: bool = False
     """Flag to check if the peers are established with negotiated AFI/SAFI. Defaults to `False`.
 
-    Can be enabled in the `VerifyBGPPeerCount` tests.
-    """
+    Can be enabled in the `VerifyBGPPeerCount` tests."""
+    redistributed_route_protocol: Redistributed_Protocol | None = None
+    """Specify redistributed route protocol."""
```

Comment:
```suggestion
    """Specify redistributed route protocol. Required field in the `VerifyBGPRedistributedRoutes` test."""
```

Also add validator in test case.